charts/consul/templates/connect-inject-mutatingwebhookconfiguration.yaml

```diff
@@ -333,6 +333,29 @@ webhooks:
       resources:
         - samenessgroups
   sideEffects: None
+{{- if (mustHas "resource-apis" .Values.global.experiments) }}
+- admissionReviewVersions:
+    - v1beta1
+    - v1
+  clientConfig:
+    service:
+      name: {{ template "consul.fullname" . }}-connect-injector
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-v2alpha1-trafficpermissions
+  failurePolicy: Fail
+  name: mutate-trafficpermissions.auth.consul.hashicorp.com
+  rules:
+    - apiGroups:
+        - auth.consul.hashicorp.com
+      apiVersions:
+        - v2alpha1
```

Comment:
🚗 this should be v2beta1 once https://github.com/hashicorp/consul/pull/18930 is merged

Reply:
yup!! i have incorporated those changes into this PR to ease the eventual rebase